pkg/kubewarden/edit/policies.kubewarden.io.policyserver.vue

```diff
@@ -52,7 +53,17 @@ export default {
   },
 
   methods: {
+    handleValidationPassed(val) {
+      this.validationPassed = val;
+    },
     async finish(event) {
+      // here we cleanup the securityContexts "seccompProfile" property if there are no keys set on the object
+      ['pod', 'container'].forEach((type) => {
+        if (this.chartValues?.spec?.securityContexts?.[type]?.seccompProfile && !Object.keys(this.chartValues?.spec?.securityContexts?.[type]?.seccompProfile).length) {
+          delete this.chartValues?.spec?.securityContexts?.[type]?.seccompProfile;
+        }
+      });
```

Comment:
We have a [utility function](https://github.com/rancher/kubewarden-ui/blob/998935a74d10044d50b7487de4b9e09de0811ebd/pkg/kubewarden/utils/object.ts#L4-L16) you could use here.

Reply:
Your's is a recursive function to cleanup all of the object properties in a given object whereas this particular place is only to clean this particular prop `seccompProfile` given a set of specific conditions. I can't use your method to clean just this one. Let me know if you disagree mate.